### PR TITLE
triagebot exclude_labels -> exclude_titles

### PR DESCRIPTION
**Pull request by pitaj**
https://github.com/rust-lang/triagebot/pull/1720

### CODE REVIEW

File: miri-script/src/commands.rs [hunk 1]
```diff
@@ -339,9 +339,9 @@ impl Command {
             "Confirmed that the push round-trips back to Miri properly. Please create a rustc PR:"
         );
         println!(
-            // Open PR with `subtree-sync` label to satisfy the `no-merges` triagebot check
+            // Open PR with `subtree update` title to satisfy the `no-merges` triagebot check
```
Comment (RalfJung) [inline miri-script/src/commands.rs:342]:
```suggestion
            // Open PR with `subtree update` title to silence the `no-merges` triagebot check
```

**Comment by RalfJung** [comment]
Thanks!
Does this have to wait until https://github.com/rust-lang/triagebot/pull/1720 lands and gets deployed? (Not sure how triagebot deals with unknown options.)

**Comment by pitaj** [comment]
It does not need to wait. Extra options in the config are ignored.

**Pull request merged**